src/slacko.ml

```diff
@@ -584,10 +584,13 @@ type history_result = [
   | timestamp_error
 ]
 
-let base_url = "https://slack.com/api/"
+let base_url = ref "https://slack.com/api/"
```

Comment:
I think we whould turn the module into a functor and split it into a inner module `Config` (though not sure what to really name it) module which contains stuff like `base_url` and an instantiation of the module with the default config module applied, so users will retain their API while we make parts modular and the test can therefore instantiate the functior with a different `Config`.

The advantages I see here is that this inner module can also hold other information like a cache of username/channel etc to it's ID and would be localized to a `base_url`. This is why I am not super sure with the name `Config` as it could be used for other useful things which do not map to a config.

Let me know what you think.